examples/README.md

```diff
@@ -71,6 +71,10 @@ under `vars` before using an example blueprint.
 
 ### (Optional) Setting up a remote terraform state
 
+There are two ways to specify terraform backends in HPC Toolkit, defaults and per-group.
```

Comment:
Make "terraform backends" a link to:

https://developer.hashicorp.com/terraform/language/settings/backends/configuration

Reply:
Done